src/olympia/lib/tests/test_cache.py

```diff
@@ -65,6 +65,15 @@ def add(*args):
     assert add(1, 2) == cache.get(cache_key)
 
 
+def test_memcached_unicode():
+    """Regression test for
+
+    https://github.com/linsomniac/python-memcached/issues/79
+    """
+    cache.set('key', u'Iñtërnâtiônàlizætiøn2')
```

Comment:
How about with an unicode key ?

Reply:
Sure can do :)

The test was already failing on 1.57 so I was happy already.